cloudify_cli/commands/summary.py

```diff
@@ -0,0 +1,104 @@
+from .. import utils
+from ..cli import cfy
+from ..table import print_data
+
+
+@cfy.group(name='summary')
+@cfy.options.common_options
+@cfy.assert_manager_active()
+def summary():
+    """Get summary information
+    """
+    pass
+
+
+@summary.command(name='nodes',
+                 short_help='Retrieve summary of node details [manager only]')
+@cfy.argument('target_field')
```

Comment:
how about `@argument(..., type=click.choice(['deployment_id']))`

Reply:
What does that gain us? I'm guessing it doesn't actually look up deployment IDs? (also, for nodes this might work currently but for node instances it can be deployment_id or node_Id, and we could end up adding other summaries to nodes and deployments later)

Reply:
validation that only the string "deployment_id" can be passed there and not other things. I mean yes, for node instances the Choice should contain more choices :P

Reply:
Ahh, OK- I'll update that tomorrow morning